CHANGELOG.md

```diff
@@ -1,3 +1,118 @@
+## v0.8.0 - 2021-05-25
+
+We're happy to announce the release of Lokomotive v0.8.0 (TODO).
```

Comment:
I think this can also be changed now?

Reply:
Oh, good. Thanks for catching that :laughing:

Reply:
All reviews are addressed.

Reply:
Done.